pynvim/api/common.py

```diff
@@ -26,7 +30,8 @@ def __init__(self, session, code_data):
         self.handle = unpackb(code_data[1])
         self.api = RemoteApi(self, self._api_prefix)
         self.vars = RemoteMap(self, self._api_prefix + 'get_var',
-                              self._api_prefix + 'set_var')
+                              self._api_prefix + 'set_var',
+                              self._api_prefix + 'del_var')
         self.options = RemoteMap(self, self._api_prefix + 'get_option',
                                  self._api_prefix + 'set_option')
```

Comment:
Should we pass an argument to `RemoteMap` to only handle specific errors as key errors?
(I mean `self.vars` should never see 'Invalid option name:')

Reply:
We probably don't need to worry about it.